### PR TITLE
Update enable-virtualization-based-protection-of-code-integrity.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the document as *Win32\_DeviceGuard* WMI class is available in Win 11 pro. 

@vinaypamnani-msft  I have not been able to check with win10 pro though. However, the user states that it appears to be present.

fixes #https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10998

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: windows/security/threat-protection/device-guard/enable-virtualization-based-protection-of-code-integrity.md [hunk 1]
```diff
@@ -205,7 +205,7 @@ Get-CimInstance –ClassName Win32_DeviceGuard –Namespace root\Microsoft\Windo
 ```
 
 > [!NOTE]
-> The *Win32\_DeviceGuard* WMI class is only available on the Enterprise edition of Windows 10 and Windows 11.
+> The *Win32\_DeviceGuard* WMI class is only available on the Professional & Enterprise edition of Windows 10 and Windows 11.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/device-guard/enable-virtualization-based-protection-of-code-integrity.md:208]:
```suggestion
> The *Win32\_DeviceGuard* WMI class is only available on the Professional and Enterprise editions of Windows 10 and Windows 11.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by vinaypamnani-msft** [comment]
It seems like the class should be available on Enterprise/Pro/Home/Education, so we should just remove this note instead.

**Comment by msbemba** [comment]
@vinaypamnani-msft Thank you for confirming. I have removed the note as per your suggestion.

Thanks
Sri

**Pull request merged**